src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,535 @@
+<div id="file-preview-card">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-preview-card can be used to display file previews. A css grid layout is used to define the areas in which certain elements can be placed.
+    </p>
+
+    <h3>Uploading</h3>
```

Comment:
To be consistent with naming of headings on site, we should go through these h3s and update them to include the name of the module:

* Uploading File Preview Card
* Image File Preview Card
* Video File Preview Card
* Document File Preview Card
* See More File Preview Card

I don't think it's necessary to have headings for the sub-examples with preview buttons. Those can just follow straight on from the previous example

The same naming conventions apply to File Preview Card Group.

Reply:
This verbose naming structure can probably be reviewed when we move to the new Skin site architecture (1 page per module)